python/race_model/EloRaceModel.py

```diff
@@ -166,6 +185,14 @@ def generatePredictions(self):
 
                     # For each matchup, calculate expected score and real score. Put results to special data structure
                     driverIds = [x["driverId"] for x in resultsForRace]
+                    # Some drivers may retire without finishing the race,
```

Comment:
I think this code should be in the Monte Carlo simulation. I think it's best to comment out this code, I'll clean it up a bit and move it there.